README.md

```diff
@@ -56,28 +56,26 @@ When you do this, great things may happen:
 
 ## Validation :heavy_check_mark:
 
-You can validate your `CITATION.cff` file with the [Python script](examples/validator.py)
-that we also use for testing the Citation File Format schema itself, like so:
+You can validate your `CITATION.cff` file on the command line with the [`cffconvert` Python package](https://pypi.org/project/cffconvert/):
 
 ```shell
-# clone this repository
-git clone https://github.com/citation-file-format/citation-file-format.git
+# Install cffconvert with pip in userspace
+python3 -m pip install --user cffconvert
 
-# change into the cloned repository directory
-cd citation-file-format
-
-# install the validation dependencies in user space
-python3 -m pip install --user ruamel.yaml jsonschema
-
-# run the validator on your CITATION.cff
-python3 examples/validator.py -s schema.json -d path/to/your/CITATION.cff
+# Validate your CFF file
+cffconvert --validate -i <path-to-your-CITATION.cff>
```

Comment:
perhaps we should leave out the `-i`, similar to how the dockerized version is explained

Reply:
Ah, I see, you don't even need the name of the file. Missed that, sorry.

```suggestion
cffconvert --validate
```